services/spar/src/Spar/Scim/Auth.hs

```diff
@@ -138,7 +140,18 @@ createScimTokenV6 ::
   -- | Request body
   CreateScimToken ->
   Sem r CreateScimTokenResponseV6
-createScimTokenV6 zusr req = responseToV6 <$> createScimToken zusr req
+createScimTokenV6 zusr Api.CreateScimToken {..} = do
```

Comment:
did we test v6 behavior as well as v7 behavior for the cases where it diverges?  don't forget to update confluence about the v7 behavior!!